### PR TITLE
Issue #32 Add Recommended Custom Instructions

### PR DESCRIPTION
**Pull request by ThirDecade2020**
Placed in README after 'Getting Started Guide' section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,12 @@ https://github.com/ykdojo/kaguya/assets/107422421/d61b8ff1-2dbd-4eb4-b1b5-45d437
 1. If you want Kaguya to be able to interact with your files, put them in the FILES folder.
 1. Note: Kaguya won't have access to files outside of its own directory.
 
+## Recommended Custom Instructions
+
+1. When editing a file of 50+ lines with Kaguya, use search and replace, not ```updateWholeFile```. Confirm with me before editing a file.
```
Comment (ykdojo) [inline README.md:26]: I think these should be in a code block format and ready for copy & pasting - what do you think?

Comment (ThirDecade2020) [reply README.md:26]: That would be better, I'll make changes and send back.

Comment (ThirDecade2020) [reply README.md]: Were you able to view the second commit ok? I changed to block format for easy copy and pasting and not sure if I re-pushed the pull request correctly the second time around. Let me know thanks!

File: README.md [hunk 2]
```diff
@@ -21,6 +21,14 @@ https://github.com/ykdojo/kaguya/assets/107422421/d61b8ff1-2dbd-4eb4-b1b5-45d437
 1. If you want Kaguya to be able to interact with your files, put them in the FILES folder.
 1. Note: Kaguya won't have access to files outside of its own directory.
 
+## Recommended Custom Instructions 
+
+```
+When editing a file of 50+ lines with Kaguya, use search and replace, not updateWholeFile. Confirm with me before editing a file. 
+When you have output from Kaguya, there's no need to repeat everything from there. Instead, you can summarize it concisely. 
+When you want to use executeCommand in a subdirectory, make sure to cd there first every time.
```
Comment (ykdojo) [inline README.md:29]:
```suggestion
When editing a file, use search and replace and NOT updateWholeFile unless we're dealing with a very small file. Confirm with me before you edit a file.

When you have output from Kaguya, there's no need to repeat everything from there. Instead, you can summarize it concisely.

When you want to use executeCommand in a subdirectory, make sure to cd there first every time.
```

**Review by ykdojo — APPROVED** [review]

**Comment by ykdojo** [comment]
Looks good! Thank you

**Pull request merged**